odoo_module_migrate/migration.py

```diff
@@ -89,9 +90,26 @@ def __init__(
         for module_name in module_names:
             self._module_migrations.append(ModuleMigration(self, module_name))
 
+        if os.path.exists(".pre-commit-config.yaml"):
+            self._run_pre_commit(module_names)
+
         # get migration scripts, depending to the migration list
         self._get_migration_scripts()
 
+    def _run_pre_commit(self, module_names):
+        logger.info("Run pre-commit")
+        _execute_shell(
+            "pre-commit run -a", path=self._directory_path, raise_error=False)
+        logger.info("Add and commit change done by pre-commit")
+        _execute_shell("git add -A", path=self._directory_path)
+        if self._commit_enabled:
+            diff = _execute_shell("git diff --cached")
+            if diff:
+                _execute_shell(
+                    "git commit -m '[IMP] %s: black, isort, prettier'  --no-verify"
```

Comment:
```suggestion
                    "git commit -m '[IMP] %s: pre-commit run'  --no-verify"
```

There are other stuff in there, such as autoflake, and it might grow or change in the future. Just say you're running pre-commit and all will be ok forever.

Reply:
If there's more stuff in the future, the message will change, but now the official message in the migration guide is that one: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0, so please leave it as is.

Reply:
There's more stuff already.

Reply:
Which one? Please tell for changing then in the migration guide.

Reply:
At least: Eslint, pyupgrade, autoflake, some hooks from https://github.com/pre-commit/pre-commit-hooks, and some others from https://github.com/oca/maintainer-tools.

I wouldn't like to keep updating the guidelines as the pre-commit toolkit evolves, so that's why I prefer the more generic message, if possible.